portlet-tck_3.0/driver/pom.xml

```diff
@@ -662,6 +662,14 @@
                            <value>${test.browser.webDriver}</value>

```

Comment:
@vsingleton Also need to introduce a property named "test.browser.headless" similar to the "integration.browser.headless" that we have in liferay-faces-test-selenium.